LNPopupController/LNPopupController/Private/LNPopupController.m

```diff
@@ -447,9 +447,10 @@ - (void)_popupBarPresentationByUserPanGestureHandler:(UIPanGestureRecognizer*)pg
 			BOOL panThreshold = CACurrentMediaTime() - _lastSeenMovement <= LNPopupBarGesturePanThreshold;
 			BOOL heightTreshold = [self _percentFromPopupBar] > LNPopupBarGestureHeightPercentThreshold;
 			BOOL isPanUp = [pgr velocityInView:_containerController.view].y < 0;
+			BOOL isPanDown = [pgr velocityInView:_containerController.view].y > 0;
 			BOOL hasPassedOffset = [pgr translationInView:_popupBar.superview].y <= LNPopupBarGestureSnapOffset;
```

Comment:
This isn't quite right here - still need to fix this line. It's always true in the case of swiping up.